src/utils/types.js

```diff
@@ -1,6 +1,8 @@
 export default {
```

Comment:
These colors should be declared globally to be consistent across the program

Reply:
The colors are defined in this separate file for now to be included anywhere. Where exactly this file resides is debatable, yet is not matter of this pull request. These definitions existed before. You could create a pull request yourself or an issue.